tests/integration/cgroups.bats

```diff
@@ -119,6 +119,59 @@ function setup() {
 	#
 }
 
+@test "runc exec (cgroup v2 without chown-cgroup annotation) does not chown cgroup" {
+	requires root cgroups_v2 systemd
+
+	set_cgroups_path
+	set_cgroup_mount_writable
+
+	# configure a user namespace
+	update_config '.linux.namespaces += [{"type": "user"}]'
+	update_config '.linux.uidMappings += [{"hostID": 100000, "containerID": 0, "size": 65536}]'
+	update_config '.linux.gidMappings += [{"hostID": 100000, "containerID": 0, "size": 65536}]'
+
+	# chown test temp dir to allow host user to read it
+	chown 100000 "$ROOT"
+
+	# chown rootfs to allow host user to mkdir mount points
+	chown 100000 "$ROOT"/bundle/rootfs
+
+	runc run -d --console-socket "$CONSOLE_SOCKET" test_cgroup_chown
+	[ "$status" -eq 0 ]
+
+	runc exec test_cgroup_chown sh -c "ls -ld /sys/fs/cgroup | awk '{print \$3}'"
```

Comment:
nit: no need to escape `$` since it's inside single quotes.

Reply:
It is inside single quotes... inside double quotes :)  So the escape is necessary.  For example:
```shell
% echo "'$0'"
'zsh'
```

Reply:
You could also use `stat`. Perhaps a bit more simple than ls + awk.

```shell
stat -c "%u %g" "/sys/fs/cgroup"
[ "$output" = "0 0" ]
```